pkg/graveler/testutil/graveler_mock.go

```diff
@@ -37,3 +39,9 @@ func InitGravelerTest(t *testing.T) *GravelerTest {
 
 	return test
 }
+
+// ShortenBranchUpdateBackOff upgrade graveler branch update back-off to shorten test duration
```

Comment:
Found out the graveler tests were way too slow - 
Ended up adding this function to this PR.
Probably belongs to a separate one, but I think we can handle this.

(Anyway, this cuts running time from 90 seconds to only 8.)